src/utils/rowTemplateColumns.ts

```diff
@@ -0,0 +1,359 @@
+      },
+      {
+        key: 'comparisonSign',
+        gridWidth: '0.2fr',
+      },
+      {
+        name: 'New',
+        key: 'new',
+        gridWidth: tableMannWhitneyConfigTestSubtestDiff.newGridWidth,
+        tooltip: 'A summary of all values from New runs using a mean.',
+      },
+      {
+        name: 'Status',
+        filter: true,
+        key: 'status',
+        gridWidth: '1fr',
+        possibleValues: [
+          { label: 'No changes', key: 'none' },
+          { label: 'Improvement', key: 'improvement' },
+          { label: 'Regression', key: 'regression' },
+        ],
+        matchesFunction(result, valueKey) {
+          switch (valueKey) {
+            case 'improvement':
+              return result.direction_of_change === 'improvement';
+            case 'regression':
+              return result.direction_of_change === 'regression';
+            default:
+              return (
+                !result.direction_of_change ||
+                result.direction_of_change === 'no change'
+              );
+          }
+        },
+        tooltip:
+          'Combination of if effect is meaningful (changes are small or greater), direction of change, and if significant',
+      },
+      {
+        name: "Cliff's Delta",
+        key: 'delta',
+        gridWidth: tableMannWhitneyConfigTestSubtestDiff.cliffsDeltaGridWidth,
+        sortFunction(resultA, resultB) {
+          return (
+            Math.abs(resultA.cliffs_delta) - Math.abs(resultB.cliffs_delta)
+          );
+        },
+        tooltip:
+          'Cliff’s Delta quantifies the magnitude of the difference between Base and New values.',
+      },
+      {
+        name: 'Significance',
+        key: 'significance',
+        filter: true,
+        gridWidth: tableMannWhitneyConfigTestSubtestDiff.significanceGridWidth,
+        tooltip:
+          'Mann Whitney U test p-value indicating statistical significance. Mann Whitney U p-value < .05 indicates a statistically significant difference between Base and New.',
+        possibleValues: [
+          { label: 'Significant', key: 'significant' },
+          { label: 'Not Significant', key: 'not significant' },
+        ],
+        matchesFunction(result, valueKey) {
+          switch (valueKey) {
+            case 'none':
+              return !result.mann_whitney_test?.interpretation;
+            default: {
+              const label = this.possibleValues.find(
+                ({ key }) => key === valueKey,
+              )?.label;
+              return result.mann_whitney_test?.interpretation === label;
+            }
+          }
+        },
+        sortFunction(
+          resultA: MannWhitneyResultsItem,
+          resultB: MannWhitneyResultsItem,
+        ) {
+          if (
+            !resultA.mann_whitney_test?.pvalue ||
+            !resultB.mann_whitney_test?.pvalue
+          ) {
+            return 0;
+          } else {
```

Comment:
These else cases (here and line 331) seem redundant to me unless I'm missing something? They could be done in a  similar manner to the `if` statement I suggested below since this return is the last line of code for the function.